packages/utilities/src/lib/pickRandom.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Picks a random element from an array
+ * @param array The array to pick a random element from
+ */
+export function pickRandom<T>(array: readonly T[], count?: 1): T;
+export function pickRandom<T>(array: readonly T[], count: number): T[];
+export function pickRandom<T>(array: readonly T[], count = 1): T | T[] {
```

Comment:
Sorry for not saying this in 1 review, but I realized this method can return the same elements multiple times. Maybe doing something like this will be a better approach
https://github.com/discordjs/discord.js/blob/main/packages/collection/src/collection.ts#L179

Reply:
Agreed. And sticking to their impl is a good idea anyway as that's been pretttyyyyy much proven to be good.